tools/csv-merger/csv-merger.go

```diff
@@ -35,19 +35,20 @@ var (
-	rookCsiResizerImage      = flag.String("rook-csi-resizer-image", "", "optional - defaults version supported by rook will be started if this is not set.")
-	rookCsiProvisionerImage  = flag.String("rook-csi-provisioner-image", "", "optional - defaults version supported by rook will be started if this is not set.")
-	rookCsiSnapshotterImage  = flag.String("rook-csi-snapshotter-image", "", "optional - defaults version supported by rook will be started if this is not set.")
-	rookCsiAttacherImage     = flag.String("rook-csi-attacher-image", "", "optional - defaults version supported by rook will be started if this is not set.")
-	noobaaContainerImage     = flag.String("noobaa-image", "", "noobaa operator container image")
-	noobaaCoreContainerImage = flag.String("noobaa-core-image", "", "noobaa core container image")
-	noobaaDBContainerImage   = flag.String("noobaa-db-image", "", "db container image for noobaa")
-	ocsContainerImage        = flag.String("ocs-image", "", "ocs operator container image")
-	ocsMustGatherImage       = flag.String("ocs-must-gather-image", "", "ocs-must-gather image")
+	rookContainerImage               = flag.String("rook-image", "", "rook operator container image")
+	cephContainerImage               = flag.String("ceph-image", "", "ceph daemon container image")
+	rookCsiCephImage                 = flag.String("rook-csi-ceph-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	rookCsiRegistrarImage            = flag.String("rook-csi-registrar-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	rookCsiResizerImage              = flag.String("rook-csi-resizer-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	rookCsiProvisionerImage          = flag.String("rook-csi-provisioner-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	rookCsiSnapshotterImage          = flag.String("rook-csi-snapshotter-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	rookCsiAttacherImage             = flag.String("rook-csi-attacher-image", "", "optional - defaults version supported by rook will be started if this is not set.")
+	noobaaContainerImage             = flag.String("noobaa-image", "", "noobaa operator container image")
+	noobaaCoreContainerImage         = flag.String("noobaa-core-image", "", "noobaa core container image")
+	noobaaDBContainerImage           = flag.String("noobaa-db-image", "", "db container image for noobaa")
+	ocsContainerImage                = flag.String("ocs-image", "", "ocs operator container image")
+	ocsMustGatherImage               = flag.String("ocs-must-gather-image", "", "ocs-must-gather image")
+	volumeReplicationControllerImage = flag.String("vol-repl-image", "", "volume replication operator container image")
```

Comment:
Lets change a flag from `"vol-repl-image"` to `"vol-replication-image"`

Reply:
I'm a little indifferent here, since this is mostly an internal too. Somehow I would say either keep it short as is or to the full mile to `volume-replication-image`....

Reply:
I'd like to keep it short (as is).